<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MorphClass</class>
 <widget class="QMainWindow" name="MorphClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>1024</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Morph</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1247</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMorphing">
    <property name="title">
     <string>Morphing</string>
    </property>
    <addaction name="actionBFS"/>
    <addaction name="actionBFS2"/>
    <addaction name="actionBFSMulti"/>
    <addaction name="actionBFSProp"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionZoomIn"/>
    <addaction name="actionZoomOut"/>
   </widget>
   <addaction name="menuMorphing"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNearestNeighbor">
   <property name="text">
    <string>Nearest Neighbor</string>
   </property>
  </action>
  <action name="actionNearestNeighborConnectivity">
   <property name="text">
    <string>Nearest Neighbor with Connectivity</string>
   </property>
  </action>
  <action name="actionBFS">
   <property name="text">
    <string>BFS 1-1</string>
   </property>
  </action>
  <action name="actionMTT">
   <property name="text">
    <string>MTT</string>
   </property>
  </action>
  <action name="actionBFS2">
   <property name="text">
    <string>BFS N-N</string>
   </property>
  </action>
  <action name="actionBFSProp">
   <property name="text">
    <string>BFS Proportional</string>
   </property>
  </action>
  <action name="actionBFSMulti">
   <property name="text">
    <string>BFS Multi Roots</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="text">
    <string>Zoom in</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="text">
    <string>Zoom out</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Morph.qrc"/>
 </resources>
 <connections/>
</ui>
